proposals/description/fip_24.txt

```diff
@@ -0,0 +1,21 @@
+FIP-24 Gas Refund for TribalChief Transition
+
+This proposal will provide gas refunds in the form of FEI to eligible addresses.
```

Comment:
Add a note saying this just funds the multisig to carry out the refunds

Reply:
Done.